packages/core/src/App/AppContent.tsx

```diff
@@ -7,6 +7,8 @@ import { getAppId, LocalStore, useIsMounted } from '@deriv/shared';
 import { observer, useStore } from '@deriv/stores';
 import { getLanguage } from '@deriv/translations';
 import { Analytics } from '@deriv-com/analytics';
+import { GrowthBook } from '@growthbook/growthbook-react';
```

Comment:
@yauheni-deriv since you are only importing `GrowthBook` class from `@growthbook/growthbook-react` , you don't need to install this package separately, because this instance can be used with our analytics package as well (details on the next comment)